drake/examples/Pendulum/Pendulum.h

```diff
@@ -0,0 +1,41 @@
+#ifndef _PENDULUM_H_
+#define _PENDULUM_H_
+
+#include <iostream>
+#include "DrakeSystem.h"
+
+using namespace std;
+using namespace Eigen;
```

Comment:
I think we should really keep it a hard rule not to use `using namespace` in headers. Just use namespace qualifiers for variables in the header if you don't want to make a .cpp file, or a `using namespace` inside function bodies. There aren't that many variables/functions from std or Eigen in this file anyway.

I can definitely imagine use cases where Pendulum.h would be included by something else.